### PR TITLE
Support for npm2yarn docs config

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added support for npm2yarn docs config

#19

### CODE REVIEW

File: docs/client-sdk/getting-started.md [hunk 1]
```diff
@@ -14,13 +15,13 @@ You can interactively use the Verida client library in your browser using the [V
 
 Start by installing the Verida client protocol library and Verida Vault authentication method:
 
-```bash
+```bash npm2yarn
 yarn add @verida/client-ts @verida/account-web-vault
```
Comment (nick-verida) [inline docs/client-sdk/getting-started.md:19]: Need the npm commands!

File: docs/client-sdk/getting-started.md [hunk 2]
```diff
@@ -14,13 +15,13 @@ You can interactively use the Verida client library in your browser using the [V
...
 ```
 
 You may receive compilation warnings when using typescript regarding `pouchdb`. If that happens, add the `pouchdb-core` type definitions:
 
-```bash
+```bash npm2yarn
 yarn add --dev @types/pouchdb-core
 ```
```
Comment (nick-verida) [inline docs/client-sdk/getting-started.md:26]: npm command

File: docs/single-sign-on-sdk/how-to-use-the-sso-sdk.md [hunk 3]
```diff
@@ -13,7 +14,7 @@ This easy to use integration method allows a user to scan a QR code to sign into
 
 This requires installing the `@verida/account-web-vault` dependency:
 
-```bash
+```bash npm2yarn
 yarn add @verida/account-web-vault
 ```
```
Comment (nick-verida) [inline docs/single-sign-on-sdk/how-to-use-the-sso-sdk.md:19]: npm command

**Review by nick-verida — CHANGES_REQUESTED** [review]
The npm2yarn thing works, but we need the npm commands there too!

File: docs/client-sdk/getting-started.md (hunk 1 above)
Comment (cmcWebCode40) [reply docs/client-sdk/getting-started.md]: @nick-verida , I have resolved this issue , you can check now

**Review by nick-verida — APPROVED** [review]
looks good, merging

**Pull request merged**